### ISSUE TITLE
PDF

### ISSUE THREAD

**Issue by yogurtnaturalny** [mailwatch/1.2.0#400]
Hi Mates 

I could not find "suggestion" tab so I will put this here

It is possible to add in version 1.3 option to create a pdf/print style report about spam ( when you go to see details about message )

I'm not so good with php but maybe you know how to pull this off ? 

Big thx for all your work and I look forward to hearing from you

**Comment by Skywalker-11** [comment]
Would this be ok? Then I can make a patch
![zwischenablage01](https://cloud.githubusercontent.com/assets/11180862/22055428/6130a4c6-dd5b-11e6-8d57-374159b1099f.jpg)

Just need to add following css and apply `class="noprint"` to all parts that shouldn't be displayed when printing. For this page its 2 elements (logo and navigation bar).
```
@media print {
        .noprint {
                display: none;
        }
        .TD.heading-w175 {
                background: transparent;
        }
}
```

**Comment by yogurtnaturalny** [comment]
This is what I was looking for, mate. A small patch but huge change :-D

**Comment by endelwar** [comment]
merged with #407
